### ISSUE TITLE
app-route appears to hijack links with `target` attributes set on them

### ISSUE THREAD

**Issue by arthurevans** [PolymerElements/app-route#105]
### Description

Go to:

https://www.polymer-project.org/1.0/blog/routing

Click on the "Open in new window" links below the demos. Note that the links do not open in a new window. (Ignore the fact that the URLs are kind of messed up.)

### Expected outcome

Links open in new window/tab.

**Comment by arthurevans** [comment]
Update: works as expected if the URL includes the host. So:

```
<a href="//www.polymer-project.org/1.0/blog/routing/demo1" target="_blank">Open in new page</a>
```

Appears the fix the problem completely. So, this is either not-an-issue or a gotcha for the documentation.

**Comment by rictic** [comment]
Fixed by https://github.com/PolymerElements/iron-location/pull/37 which I just released as v0.8.3 of iron-location.
